src/components/VideoPlayerPreview/index.tsx

```diff
@@ -12,30 +13,21 @@ import useWindowDimensions from '@hooks/useWindowDimensions';
 import CONST from '@src/CONST';
 import VideoPlayerThumbnail from './VideoPlayerThumbnail';
 
-const propTypes = {
-    videoUrl: PropTypes.string.isRequired,
-
-    videoDimensions: PropTypes.shape({
-        width: PropTypes.number.isRequired,
-        height: PropTypes.number.isRequired,
-    }),
-
-    videoDuration: PropTypes.number,
-
-    thumbnailUrl: PropTypes.string,
-
-    fileName: PropTypes.string.isRequired,
-
-    onShowModalPress: PropTypes.func.isRequired,
+type VideoDimensions = {
+    width: number;
+    height: number;
 };
 
-const defaultProps = {
-    videoDimensions: CONST.VIDEO_PLAYER.DEFAULT_VIDEO_DIMENSIONS,
-    thumbnailUrl: undefined,
-    videoDuration: 0,
+type VideoPlayerPreviewProps = {
+    videoUrl: string;
+    videoDimensions: VideoDimensions;
+    videoDuration: number;
```

Comment:
```suggestion
    videoDuration?: number;
```

According to propTypes